sources/settings-extensions/ntp/src/lib.rs

```diff
@@ -85,4 +87,22 @@ mod test {
         let results = serde_json::to_string(&ntp).unwrap();
         assert_eq!(results, test_json);
     }
+
+    fn test_options_ntp() {
+        let test_json = r#"{"time-servers":["https://example.net","http://www.example.com"],"options": ["minpoll", "1", "maxpoll", "2"]}"#;
+
+        let ntp: NtpSettingsV1 = serde_json::from_str(test_json).unwrap();
+        assert_eq!(
+            ntp.options.clone().unwrap(),
+            vec!(
+                Url::try_from("minpoll").unwrap(),
+                Url::try_from("1").unwrap(),
+                Url::try_from("maxpoll").unwrap(),
+                Url::try_from("2").unwrap(),
```

Comment:
These should be `String`s, not `Url`s:

```suggestion
                String::from("minpoll"),
                String::from("1"),
                String::from("maxpoll"),
                String::from("2"),
```